src/js/utils/Select.jsx

```diff
@@ -35,11 +34,11 @@ class Select extends Component {
   handleChange(value) {
     if (this.props.multi) {
       const values = _.map(value, val =>
-        (this.props.objectValue ? JSON.parse(val.value) : val.value));
+        (this.props.objectValue ? val : val));
       this.setState({ value: values });
       this.props.onChange(values);
     } else if (value !== null && value !== undefined) {
-      const val = this.props.objectValue ? JSON.parse(value.value) : value.value;
+      const val = this.props.objectValue ? value.value : value;
```

Comment:
you don't have to check the objectValue here

Reply:
yup, agree